spec/helpers/events_helper_spec.rb

```diff
@@ -11,5 +11,4 @@
 #   end
 # end
 RSpec.describe EventsHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You did none of the two options that this line gave you. Are you sure that having an empty spec is good?